src/store/message.ts

```diff
@@ -50,5 +53,17 @@ export const useMessageStore = create<MessageState>((set) => ({
       newPreviewMessages.set(key, message);
       return { previewMessages: newPreviewMessages };
     }),
-  setPreviewMessages: (previewMessages) => set(() => ({ previewMessages }))
+  setPreviewMessages: (previewMessages) => set(() => ({ previewMessages })),
+  selectedProfile: '',
+  setSelectedProfile: (selectedProfile) => set(() => ({ selectedProfile })),
+  reset: () =>
+    set((state) => {
```

Comment:
Very neat that we don't have to reset the `client` here since the address doesn't change, however -- do we still reset the client properly when logging out? I think we have a `reset()` inside `useXmtpClient` but want to make sure we don't regress there.

Reply:
I think we have removed reset from there too. I guess we wipe keys on disconnect and set the client to undefined there